src/structures/member.ts

```diff
@@ -30,6 +30,7 @@ export class Member extends SnowflakeBase {
   deaf!: boolean
   mute!: boolean
   guild: Guild
+  /** Use `edit` method to update permissions */
```

Comment:
Editing permissions of a member itself is not possible. These are just calculated permissions using roles.

Reply:
Used find & replace, mb